README.md

```diff
@@ -30,18 +30,20 @@ Projects listed on this page are open source that Microsoft maintain or contribu
+|                                      | [Eraser](https://github.com/Azure/eraser) <br> ([releases](https://github.com/Azure/eraser/releases))                                                                                                                                                                              | Cleaning up images from Kubernetes nodes                                                                                                                                                                                                                                      | Azure: incubation <br> API: v1                                                                                                                                                                         | [GitHub issues](https://github.com/Azure/eraser/issues)                                                                                                                                                                                                                                                                                                                                        | [How to use (OSS)](https://github.com/Azure/eraser#getting-started) <br> [How to use (AKS)](https://learn.microsoft.com/en-us/azure/aks/image-cleaner?tabs=azure-cli)                                                                                                                                                         |
+| Container Security                   |                                                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                               |                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Does this need to be a separate category? or could it be under `Cloud Native Governance and Security` like all the other security projects?

Reply:
IMO I think `Cloud Native` is generic enough so we can reuse the same category.

Reply:
updated